src/styles/utilis/_variables.scss

```
@@ -0,0 +1,15 @@
+$color-blue:#00acdc;
+$color-white: #fff;
+$text-gray: #616070;
+$text-dark: #060b35;
+
+$default-font-family: roboto, helvetica, sans-serif;
+$default-font-size: 10px;
+$default-line-height: 14px;
+
+$thinner-font-weight: 500;
+$bolder-font-weight: 700;
```

please just add font-weight to variable name 
```suggestion
$font-weight-500: 500;
$font-weight-700: 700;
```